Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs

```diff
@@ -259,9 +262,6 @@ private Ray SmoothGaze(Ray? newGaze)
                     isSaccading = true;
                 }
 
-                Vector3 v1 = oldGaze.Value.origin + oldGaze.Value.direction;
-                Vector3 v2 = newGaze.Value.origin + newGaze.Value.direction;
-
```

Comment:
Were these values not used anywhere and that is why they were removed?

Reply:
Yep! Oddly, they were calculated but the resulting vectors weren't actually used. I tried to walk through the history a bit, but couldn't find a point where they were ever used.